.github/workflows/ci.yml

```diff
@@ -241,7 +241,7 @@ jobs:
           -DUSE_CCACHE=ON \
           -DBUILD_DOCUMENTATION=YES \
           -DCMAKE_BUILD_TYPE=${{ matrix.ci.build_type }} ..
-        make -j 2
+        make -j $(nproc)
```

Comment:
```suggestion
        cmake --build . --parallel
```

Reply:
I've some bad experience of "make -j" without values on directories with a very large number of .cpp files where it forks a gcc for each file, which exhausts the RAM and in the best case trigger OOM killer and in the worse case forces me to reboot...

Reply:
Yup, on second thought I've had these issues too (and I normally hardcode this number too, leaving "1 cpu" remaining to "do other things"). In this case, you may reject my suggestions.